Stylesheets/commons/utils.css

```diff
@@ -19,3 +19,11 @@
 .gap-sm{
     gap:1rem;
 }
+
+.dis-flex{
+    display:flex
+}
+
+.flex-sb{
+    justify-content: space-between;
+}
```

Comment:
instead of `flex-sb` you can name it like `justify-content-sb`, because `flex-sb` doesn't specifies which property is this class referring to since flex has many properties that take the same value of `space-between`.